pkg/machines/components/vm/disks/diskAdd.jsx

```diff
@@ -366,10 +397,11 @@ export class AddDiskModalBody extends React.Component {
                     if (poolName)
                         stateDelta = { ...stateDelta, ...this.existingVolumeNameDelta(this.getDefaultVolumeName(poolName), prevState.storagePoolName) };
                 }
+                if (value === CUSTOM_PATH) // user moved to CUSTOM_PATH subtab
```

Comment:
I think it's cleaner to always assign 'stadeDelte.mode = value' on the top of this case, and then do the per mode modifications.